features/src/kafka_util.py

```diff
@@ -0,0 +1,73 @@
+# Copyright © 2021 Pavel Tisnovsky, Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""An interface to Apache Kafka done using kcat utility and Python interface to Kafka."""
+
+
+import subprocess
+import json
+
+from kafka.admin import NewTopic
+from kafka import KafkaProducer, KafkaConsumer
+from kafka import KafkaAdminClient
+from kafka.errors import UnknownTopicOrPartitionError, TopicAlreadyExistsError
```

Comment:
Is this kafka-python library? I'm ok with it, but maybe it'll cause a unwanted change in the future, as it seems to be abandoned